### DISCUSSION TITLE
如何在英文title下插入校徽？

### DISCUSSION THREAD

**Discussion by giserzy** [whutug/whu-thesis#191] [Q&A]
如题，看到作者提供了whulogo.pdf，但是不清楚如何插入扉页，请大家指点一二。

**Comment by xkwxdyy** [comment]
你为什么要在扉页加logo

**Reply by note286** [reply]
我猜测是这位同学觉得有logo好看，或者觉得就应该有logo，是开发者遗漏了。

**Reply by xkwxdyy** [reply]
开发都是基于教务处的规范而来的，你如果找到扉页要加logo的说明，请发出来。

**Reply by giserzy** [reply]
嗯嗯嗯
